README.md

```diff
@@ -308,6 +308,127 @@ for vehicle in vehicle_path_list:
 
 All of the simulations will be launched in one go. The user can check the status of the simulations and retrieve the results when they are ready. Check the FAQ section for more information on how to do this.
 
+## Task management
+
+
+### Using the `Task` class
+
+As you may have noticed, both the sync and async runs of the scenarios show above return a `Task` object to the user.
+This object provides several methods to manage a task submitted to the **Inductiva API**.
+For instance, you can get information about the task (its current status, when it started, information on the machine it is running, ...), kill it while
+it is still running if you've changed your mind, or download its output files (all of them or only those you need) after it completed.
+
+Check out some examples after submitting an asynchronous simulation:
+
+```python
+# `scenario`` constructed as in the examples above.
+# ... stands for arguments related to each specific scenario.
+task = scenario.simulate(..., run_async=True)
+
+# Kill a task that is running in the API.
+task.kill()
```

Comment:
I'd show before how to get the status of a task and simulate that it is running. Then I'd kill it. This way you don't have to redefine `task` in the documentation again and again.

Reply:
i still need to do it for the download part, because it doesn't make much sense to exemplify the downloads after `killing()`